test/kubeclient_test.rb

```diff
@@ -1,7 +1,4 @@
-require 'minitest/autorun'
-require 'json'
-require 'webmock/minitest'
-require './lib/kubeclient'
+require './test/test_helper'
```

Comment:
@Fryguy 
it doesn't work without.
I tried "test/test_helper" or just "test_helper"
when doing rake test it gets "cannot load such file -- test/test_helper" error